app/views/catalog/_show_sidebar.html.erb

```diff
@@ -2,6 +2,23 @@
 	<%= render :partial => 'show_tools' %>
 <% end %>
 
+<% if @document.research_help %>
+	<%= render partial: "show_research_help" %>
+<% end %>
+
+<% if @document.active_fedora_model == 'Item' %>
+<div class="text-muted text-uppercase small strong">Collection</div>
+  <h4><%= render_document_show_field_value @document, field: Ddr::IndexFields::IS_MEMBER_OF_COLLECTION %></h4>
+  <p><%= truncate((parent_abstract @document["is_member_of_collection_ssim"]), :length => 180, :separator => ' ') %></p>
```

Comment:
@seanaery Relatively minor point but I'll mention it ... Ddr::IndexFields::IS_MEMBER_OF_COLLECTION (in line 11 above) is a constant that contains the string "is_member_of_collection_ssim"; i.e., it is a constant that holds the actual name of the Solr index key.  (The same is true of all the Ddr::IndexFields::... constants.)  So, in lines 11 and 12, you refer to the same piece of data -- "is_member_collection_ssim" -- two different ways: using the constant (line 11) and using the string directly (line 12).  It's better style, I think, to be consistent, especially since the lines are side by side.  And I would opt for using the constant in both cases; i.e., line 12 would contain `@document[Ddr::IndexFields::IS_MEMBER_OF_COLLECTION]` instead of what you have now.